### DISCUSSION TITLE
Alias with multiple values to resolve

### DISCUSSION THREAD

**Discussion by niksy** [rolldown/rolldown#3057] [General]
Will Rolldown support defining multiple values for alias?

It seems like oxc-resolver is compatible with Webpack’s enhanced-resolve which already supports that.

There was a similar [discussion](https://github.com/vitejs/vite/issues/17423) about this but I don’t know if scope has changed in regards to this functionality.

**Comment by IWANABETHATGUY** [comment]
It is already supported in rust side, i updated api with https://github.com/rolldown/rolldown/pull/3060 in js side.

**Comment by niksy** [comment]
Thank you, this is excellent news!
